tools/build_defs/repo/utils.bzl

```diff
@@ -119,7 +118,7 @@ def download_remote_files(ctx, auth = None):
     for p in pending:
         p.wait()
 
-def patch(ctx, patches = None, patch_cmds = None, patch_cmds_win = None, patch_tool = None, patch_args = None, auth = None):
```

Comment:
Is this a breaking change?

Reply:
Nice catch, I added it back as a way to override the auth by user.